2020/25xxx/CVE-2020-25685.json

```diff
@@ -1,18 +1,86 @@
 {
```

Comment:
CNA(s) of user (cisco) do not match owning CNA.

Reply:
ACK. I thought that I could update the CVE. I will close the pull request and wait until the CNA for `dnsmasq` submits the CVE. I am not sure who that would be, but I will stand by. ;-) I am very sure that several other vendors will also be affected.